c7n/resources/rds.py

```diff
@@ -1308,12 +1308,101 @@ class resource_type(object):
         dimension = None
         date = None
 
+    def augment(self, resources):
+        _db_subnet_group_tags(
```

Comment:
there's no point for the augment or the helper method on an action.

Reply:
i guess the question is why is this being done in a delete action?

Reply:
Hi, after testing i found that the tags on the rds subet group resource weren't working, so I added the augment on the RDSSubnetGroup QueryResourceManager class, and subsequent helper method, I observed this pattern in other places in the codebase, can you point me to a more preferable pattern. Thanks A.

Reply:
sorry the web diff view made it a little confusing where the augment was going, i checked it out locally and the usage here looks good.